frontend/src/store/mutations.js

```diff
@@ -12,4 +13,7 @@ export const mutations = {
   [SET_LOGIN_ERROR](state, loginError) {
     Object.assign(state, { loginError });
   },
+  [SET_UPDATE_READY](state, isReady) {
+    Object.assign(state, { updateReady: isReady })
```

Comment:
I will enable linter for Vue to find small issues like that one (missing coma).

Reply:
this all comes down to style. I am happy to add coma though.